src/main/java/com/shootdoori/match/entity/lineup/Lineup.java

```diff
@@ -0,0 +1,138 @@
+package com.shootdoori.match.entity.lineup;
+
+import com.shootdoori.match.entity.match.Match;
+import com.shootdoori.match.entity.match.request.MatchRequest;
+import com.shootdoori.match.entity.match.waiting.MatchWaiting;
+import com.shootdoori.match.entity.team.TeamMember;
+import com.shootdoori.match.entity.user.UserPosition;
+import jakarta.persistence.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "lineup")
+public class Lineup {
```

Comment:
```suggestion
public class Lineup extends DateEntity {
```

DateEntity가 추가된지 좀 오래된 것 같은데요..?